pkg/queryexpr/query_expr.go

```diff
@@ -0,0 +1,98 @@
+package queryexpr
```

Comment:
please check again the necessities for each of the types, variable (const), method, and functions if they need to be exported. if there is no requirements from outside, please keep them unexported.

Reply:
Updated. 
Sorry I just figured out that types can set as unexported too, thank you